lib/putRuleConfigKey.js

```diff
@@ -0,0 +1,10 @@
+const apiCall = require('./api');
+
+const putRuleConfigKey = async value =>
+  apiCall({
+    path: 'rules-configs/AUTH0_ACCOUNT_LINKING_EXTENSION_CUSTOM_DOMAIN',
```

Comment:
nit: Similar commentary to [delete](https://github.com/auth0-extensions/auth0-account-link-extension/pull/204/files#r2074277316)